specs/core/0_fork-choice.md

```diff
@@ -55,11 +55,11 @@ The head block root associated with a `store` is defined as `get_head(store)`. A
 
 ### Helpers
 
-#### `Target`
+#### `Checkpoint`
 
 ```python
-@dataclass
-class Target(object):
+@dataclass(eq=True, frozen=True)
```

Comment:
This is temporary to make `Checkpoint` usable as a lookup key in a dictionary.
In #1210, we elevate `Checkpoint` to an SSZ type in beacon chain spec and this is removed